benchmark/Streamly/Benchmark/Data/Array/Foreign/Mut.hs

```diff
@@ -17,23 +17,81 @@
 {-# OPTIONS_GHC -fplugin Test.Inspection.Plugin #-}
 #endif
 
-module Main (main) where
-
 import Control.DeepSeq (NFData(..))
+import Control.Monad.IO.Class (MonadIO)
+import Data.Functor ((<&>))
 import Streamly.Prelude (MonadAsync, SerialT, IsStream)
+import Streamly.Internal.Data.Stream.Serial (SerialT(..))
 import System.Random (randomRIO)
 
-import qualified Streamly.Prelude as Stream
 import qualified Streamly.Internal.Data.Array.Foreign as Array
 import qualified Streamly.Internal.Data.Array.Foreign.Mut as MArray
+import qualified Streamly.Internal.Data.Stream.StreamD as D
+import qualified Streamly.Prelude  as S
+import qualified Streamly.Prelude as Stream
+
+import Gauge
+import Prelude as P hiding (map)
```

Comment:
import qualified, better import the full name Prelude rather than P.

Reply:
importing this `quialified` is rather tedious to work with.

Reply:
Doing it though.